docs/actions.md

```diff
@@ -523,13 +523,13 @@ Creating the zip file (example with two additional modules):
 
 Creating the action: 
 
-```$ wsk action create helloPythonZip --kind python helloPython.zip```
+```$ wsk action create helloPython --kind python helloPython.zip```
 
 ### Packaging Python actions with a virtual environment in zip files
 
-The virtual environment package *virtualenv* is supported to pass pip installed packages together with the action in the zip file.
-To ensure compatibility with the openwhisk container, package installations inside virtualenv must be done in the target environment. 
-So the docker image *openwhisk/pythonaction* is used to create the virtualenv directory. 
+The virtual environment package `virtualenv` is supported to pass pip installed packages together with the action in the zip file.
+To ensure compatibility with the OpenWhisk container, package installations inside virtualenv must be done in the target environment. 
+So the docker image `openwhisk/pythonaction` is used to create the virtualenv directory. 
```

Comment:
Adjust to reflect that you're using either `openwhisk/python{2,3}Action` respectively.